cmd/telemetrygen/internal/logs/worker_test.go

```diff
@@ -145,6 +146,13 @@ func TestLogsWithOneTelemetryAttributes(t *testing.T) {
 		for i := 0; i < rlogs.Len(); i++ {
 			attrs := rlogs.At(i).ScopeLogs().At(0).LogRecords().At(0).Attributes()
 			assert.Equal(t, 2, attrs.Len(), "shouldn't have less than 2 attributes")
+
+			val, ok := attrs.Get(telemetryAttrKeyOne)
+			assert.True(t, ok, fmt.Sprintf("there should be an attribute with key %s", telemetryAttrKeyOne))
```

Comment:
nit: You can just use [`assert.Truef`](https://pkg.go.dev/github.com/stretchr/testify/assert#Truef):
```suggestion
			assert.Truef(t, ok, "there should be an attribute with key %s", telemetryAttrKeyOne)
```